arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -13219,195 +13379,450 @@
     "SiteAuthSettings": {
       "description": "Configuration settings for the Azure App Service Authentication / Authorization feature.",
       "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
       "properties": {
-        "enabled": {
-          "description": "<code>true</code> if the Authentication / Authorization feature is enabled for the current app; otherwise, <code>false</code>.",
-          "type": "boolean"
-        },
-        "runtimeVersion": {
-          "description": "The RuntimeVersion of the Authentication / Authorization feature in use for the current app.\n The setting in this value can control the behavior of certain features in the Authentication / Authorization module.",
-          "type": "string"
-        },
-        "unauthenticatedClientAction": {
-          "description": "The action to take when an unauthenticated client attempts to access the app.",
+        "properties": {
+          "description": "SiteAuthSettings resource specific properties",
+          "properties": {
+            "enabled": {
+              "description": "<code>true</code> if the Authentication / Authorization feature is enabled for the current app; otherwise, <code>false</code>.",
+              "type": "boolean"
+            },
+            "runtimeVersion": {
+              "description": "The RuntimeVersion of the Authentication / Authorization feature in use for the current app.\n The setting in this value can control the behavior of certain features in the Authentication / Authorization module.",
+              "type": "string"
+            },
+            "unauthenticatedClientAction": {
+              "description": "The action to take when an unauthenticated client attempts to access the app.",
+              "enum": [
+                "RedirectToLoginPage",
+                "AllowAnonymous"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "UnauthenticatedClientAction",
+                "modelAsString": false
```

Comment:
Setting modelAsString to false can result in breaking changes whenever you modify this enum